### PR TITLE
Add Fig as an installation method to the README

### PR DESCRIPTION
**Pull request by ibayramli**
The [Fig Plugin Store](https://fig.io/plugins) supports 1-click install for 400+ shell plugins. We have over 100k users, thousands of whom manage their shell configuration with Fig.

Zsh Autosuggest is already listed in the store so we'd love to have it listed as a download method on your readme.

Thanks so much and please let me know if you have any questions!

### CODE REVIEW

**Comment by z0rc** [comment]
@ibayramli2001 please tone down the wording. This isn't a place to advertise Fig and its features.

Also you're using same template with advertisement text for PRs in multiple repos. Please stop using it.

**Comment by ericfreese** [comment]
Pretty much agreed with @z0rc. I'm ok with listing it here but let's skip the marketing talk and have just a link to the fig plugin page, something along [these lines](https://github.com/zsh-users/zsh-syntax-highlighting/pull/877#issuecomment-1157043600) is fine.

Out of curiosity, doesn't the functionality from the plugin kind of clash with Fig's autocompletion? Do they actually work well together?

**Comment by ibayramli** [comment]
Thanks for reviewing this @ericfreese. Although currently the button links to a plugin store webpage, it will soon redirect the link to open up zsh-autosuggestions plugin store page in desktop for Fig users. It is more clear to have a button open up a desktop app than a hyperlink, so we would strongly prefer to have a button.

Re zsh autosuggest and Fig autocomplete conflicts, I would say they complement each other rather than conflict. Here is a demo:

https://user-images.githubusercontent.com/43158991/174156245-5fc858bc-496b-4d72-8660-7126df15dcd6.mov

File: INSTALL.md [hunk 1]
```diff
@@ -28,6 +29,14 @@
 
 2. Start a new terminal session.
 
+## Fig
+
+[Fig](https://fig.io) adds apps, shortcuts, and autocomplete to your existing terminal.
+
+Install `zsh-autosuggestions` in just one click.
+
+<a href="https://fig.io/plugins/other/zsh-autosuggestions" target="_blank"><img src="https://fig.io/badges/install-with-fig.svg" /></a>
```
Comment (brendanfalk) [inline INSTALL.md:38]:
```suggestion
Click [here](https://fig.io/plugins/other/zsh-autosuggestions) to 1-click install `zsh-autosuggestions` using [Fig](https://fig.io)
```

**Review by brendanfalk — CHANGES_REQUESTED** [review]

**Comment by brendanfalk** [comment]
@ericfreese - My apologies for massive delay here (have just been cleaning out my GH notifications)

I've updated the PR factoring in you commentary above.

Intention is not to be markety or salesy. We truly do have a bunch of users, zsh-autosuggestion is incredibly popular (we feature it heavily in our store), and having us listed as an install method helps other users quickly download it. 

Apologies again for the issues above - please let me know if any more questions/concerns!

**Pull request closed without merging**